unstructured_ingest/v2/processes/connectors/astradb.py

```diff
@@ -129,11 +130,6 @@ class AstraDBIndexerConfig(IndexerConfig):
         "numbers, and underscores."
     )
     keyspace: Optional[str] = Field(default=None, description="The Astra DB connection keyspace.")
-    namespace: Optional[str] = Field(
```

Comment:
datastax is ok with us removing this?

Reply:
It's deprecated and this next bump is going to be a minor bump due to the azure ai search name change. Perfect time to remove this.